homeassistant/components/ping/binary_sensor.py

```diff
@@ -32,9 +31,7 @@
 ATTR_ROUND_TRIP_TIME_MDEV = "round_trip_time_mdev"
 ATTR_ROUND_TRIP_TIME_MIN = "round_trip_time_min"
 
-SCAN_INTERVAL = timedelta(minutes=5)
-
-PARALLEL_UPDATES = 50
+PARALLEL_UPDATES = 1
```

Comment:
I am not sure if this works when everything is run in a coordinator

Reply:
Doesn't seem to work, yes